distribution/docs/src/main/resources/_contents/_configuring/configuring-web-service-providers-contents.adoc

```diff
@@ -0,0 +1,7 @@
+
+==== Configuring ${branding} Web Service Providers
```

Comment:
Since this talks about changing a configuration file, should this be a sub-section of 3.3, or included in the description of the `hostname` attribute in the 3.3.1 section?